### DISCUSSION TITLE
SELECT array_value(1, 2, 3); error

### DISCUSSION THREAD

**Discussion by jimdickson007** [duckdb/duckdb#10331] [Q&A] [answered]
I have installed v0.9.2 and took example from v0.9.3 docs (no new features)
this is 2nd issue I have encounted with arrays (I am brand new to Duckdb) - am I missing something fundamental
or is this feature/doc bug ?

> Installs$ ./duckdb
> v0.9.2 3c695d7ba9
> Enter ".help" for usage hints.
> Connected to a transient in-memory database.
> Use ".open FILENAME" to reopen on a persistent database.
> D select * from pragma_version();
> ┌─────────────────┬────────────┐
> │ library_version │ source_id  │
> │     varchar     │  varchar   │
> ├─────────────────┼────────────┤
> │ v0.9.2          │ 3c695d7ba9 │
> └─────────────────┴────────────┘
> D SELECT array_value(1, 2, 3);
> Error: Catalog Error: Scalar Function with name array_value does not exist!
> Did you mean "any_value"?
> LINE 1: SELECT array_value(1, 2, 3);
>                ^
> D .exit

https://duckdb.org/docs/sql/data_types/array.html

> -- Construct with the 'array_value' function
> SELECT array_value(1, 2, 3);
> -- You can always implicitly cast an array to a list (and use list functions, like list_extract, '[i]')
> SELECT array_value(1, 2, 3)[2];
> -- You can cast from a list to an array, but the dimensions have to match up!
> SELECT [3, 2, 1]::INT[3];
> -- Arrays can be nested
> SELECT array_value(array_value(1, 2), array_value(3, 4), array_value(5, 6));
> -- Arrays can store structs
> SELECT array_value({'a': 1, 'b': 2}, {'a': 3, 'b': 4});

**Comment by Tishj** [comment, marked as answer]
Just as before, this page says `dev`

Feel free to check out the prerelease version, easiest way to do is using the python package:
`python3 -m pip install duckdb --pre`

**Reply by jimdickson007** [reply]
Cheers

**Comment by jimdickson007** [comment]
Closing
